lib/string.rb

```diff
@@ -2,4 +2,45 @@ class String
   def humanized_all_first_capitals
     self.humanize.split(' ').map{|w| w.capitalize}.join(' ')
   end
+  
+  def downcase_words
+    self.scan(/\b\w+\b/).map(&:downcase)
+  end
+  
+  # Methods below created to customly setup slugs from friendly_id gem
+  
+  def short_name
```

Comment:
No idea ;). We could do that, but than it would probably complicate the code for friendly_id implementation, as it's not supposed to work like that (at least according to the documentation I read).